README.md

```diff
@@ -14,44 +14,57 @@ There are various flavors of configuration that Farscape supports for entering a
 a response with a supported Hypermedia media-type and a root that lists available resources as links.
 
 ### A Hypermedia API
-For a interacting with an API (or individual service that supports a list of resources at its root), you enter the
-API and follow your nose using the `enter` method on the agent. This method returns a [Farscape::Representor]() 
-instance with a simple state-machine interface of `attributes` (data) and `transitions` (link/form affordances) for 
-interacting with the resource representations.
+For a interacting with an API (or individual service that supports a list of resources at its root), you enter the API and follow your nose using the `enter` method on the agent. This method returns a [Farscape::Representor](https://github.com/mdsol/farscape/blob/develop/lib/farscape/representor.rb) instance with a simple state-machine interface of `attributes` (data) and `transitions` (link/form affordances) for interacting with the resource representations.
```

Comment:
- that full github link should be a relative link: shorter and refers the proper file when you're on a branch
- the "This method..." sentence (and others in the same case) should be on their own line to make easier to diff/comment on things. remember that going on a new line won't put a blank line in the output so the visual effect is going to be the same as right now.